webhook/command_response.go

```diff
@@ -0,0 +1,54 @@
+package webhook
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/micromdm/micromdm/connect"
+	"github.com/micromdm/micromdm/pubsub"
+	"github.com/pkg/errors"
+)
+
+const contentType = "application/x-apple-aspen-mdm"
+
+type CommandResponse struct {
+	CallbackUrl string
+}
+
+func (r CommandResponse) StartListener(sub pubsub.Subscriber) error {
+	if r.CallbackUrl == "" {
+		return nil
+	}
+
+	connectEvents, err := sub.Subscribe(context.TODO(), "responseWebhook", connect.ConnectTopic)
+	if err != nil {
+		return errors.Wrapf(err,
+			"subscribing devices to %s topic", connect.ConnectTopic)
+	}
+
+	go func() {
+		for {
+			select {
+			case event := <-connectEvents:
+				var ev connect.Event
+				if err := connect.UnmarshalEvent(event.Message, &ev); err != nil {
+					fmt.Println(err)
+					continue
+				}
+
+				fmt.Printf("Content of event %s: %d\n", ev.Response.Status, len(ev.OriginalResponse))
+
+				c := http.Client{}
```

Comment:
By creating a new httpClient each time, you'll open a new file descriptor and run out of resources very fast. client should be a dependency of the `CommandResponse` struct, initialized in `package main` and passed down to all resources that require an HTTP Client.